### PR TITLE
Tmz 2.1.0 bugfix telegraphs prussian

### PR DESCRIPTION
**Pull request by t-m-z**
adjust output semaphore
adjust titel/description murray
adjust maps for wheatstone cooke

### CODE REVIEW

File: assets/i18n/en.json [hunk 1]
```diff
@@ -4401,24 +4401,24 @@
...
-  "telegraph_edelcrantz_a_museum_first": "FIRST",
-  "telegraph_edelcrantz_a_museum_second": "SECOND",
-  "telegraph_edelcrantz_a_museum_third": "THIRD",
-  "telegraph_edelcrantz_a_museum_former": "FORMER",
-  "telegraph_edelcrantz_a_museum_latter": "LATTER",
+  "telegraph_edelcrantz_a_museum_whoamitalkingto": " WHO AM I TALKING TO ",
+  "telegraph_edelcrantz_a_museum_tellmemore": " TELL ME MORE ",
+  "telegraph_edelcrantz_a_museum_yes": " YES ",
+  "telegraph_edelcrantz_a_museum_no": " NO ",
+  "telegraph_edelcrantz_a_museum_maybe": " MAYBE ",
```
Comment (S-Man42) [inline assets/i18n/en.json:4410]: Are you quite sure about the spaces? For me it seems like a clear formatting issue ans should be done in the code. Especially when it comes to foreign translations: I am sure nobody will translate it with spaces.

Comment (Birdy2404) [reply assets/i18n/en.json:4410]: Crowdin takes care for the spaces at the beginning and at the end. If they are in the en.json, you will get a hint that they are missing in your translation and you need to decide if you want to change your text, or if you want to save it like it is.

**Comment by t-m-z** [comment]
there was a reason for this - however, I do not remember so it was - quite sure - not a good one.
I will check and change

Sent from MailDroid

-----Original Message-----

**Comment by t-m-z** [comment]
there was a reason for this - however, I do not remember so it was - quite sure - not a good one.
I will check and change

Sent from MailDroid

-----Original Message-----

File: assets/searchstrings/common.json [hunk 2]
```diff
@@ -539,15 +539,15 @@
   "telegraph": "",
   "telegraph_gaussweber": "carlgauss johanncarlfriedrichgauss wilhelmeduardweber +-",
   "telegraph_schillingcanstatt": "paulludwigschillingvoncanstatt paulludwigschillingvoncannstatt schillingcannstatt schillingcanstatt",
-  "telegraph_ccitt": "ccitt",
+  "telegraph_ccitt": "ccitt baudot murray",
```
Comment (S-Man42) [inline assets/searchstrings/common.json:542]: Can't you reuse the ccitt searchstring?

File: assets/i18n/en.json [hunk 3]
```diff
@@ -4398,13 +4398,13 @@
...
-  "telegraph_edelcrantz_a_museum_repeatmessage": " REPEAT MESSAGE ",
-  "telegraph_edelcrantz_a_museum_endcommunication": " END COMMUNICATION ",
+  "telegraph_edelcrantz_a_museum_messagereceived": "MESSAGE RECEIVED",
+  "telegraph_edelcrantz_a_museum_doyoucopy": "DO YOU COPY?",
+  "telegraph_edelcrantz_a_museum_understood": "UNDERSTOOD",
+  "telegraph_edelcrantz_a_museum_repeatmessage": "REPEAT MESSAGE",
+  "telegraph_edelcrantz_a_museum_endcommunication": "END COMMUNICATION",
   "telegraph_edelcrantz_a_museum_whoamitalkingto": "WHO AM I TALKING TO",
-  "telegraph_edelcrantz_a_museum_tellmemore": "TELL ME MORE",
+  "telegraph_edelcrantz_a_museum_tellmemore": " TELL ME MORE",
```
Comment (S-Man42) [inline assets/i18n/en.json:4407]: Still a space left

File: assets/searchstrings/de.json [hunk 4]
```diff
@@ -507,6 +506,7 @@
   "symbol_surf": "windsurfen wellenreiten wellenreiter surfbretter surfer",
   "symbol_tae": "telekommunikationsanschlusseinheit telefon leitung kabel",
   "symbol_tamil_numerals": "zahlen ziffern indien indisch",
+  "symbol_telegraph_prussia": "preussen",
```
Comment (S-Man42) [inline assets/searchstrings/de.json:509]: Reuse telegraph_prussia here instead new identical searchstring.

Additionally keyword may be preussischer or preussisches (if someone searches for preussischer telegraf)

File: assets/searchstrings/en.json [hunk 5]
```diff
@@ -507,6 +506,7 @@
   "symbol_surf": "windsurfing surfride surfer surfriding surfride surfboard wavesurfing",
   "symbol_tae": "connector telecommunications connection unit telephone line cable",
   "symbol_tamil_numerals": "numbers numerals indian",
+  "symbol_telegraph_prussia": "prussia prussian",
```
Comment (S-Man42) [inline assets/searchstrings/en.json:509]: prussia is not required because it's contained in prussian

File: assets/searchstrings/en.json [hunk 6]
```diff
@@ -550,8 +550,8 @@
   "telegraph_ohlsen": "flaps shutter norway",
   "telegraph_prussia": "prussia",
   "telegraph_semaphore": "semaphore flags",
-  "telegraph_wigwag": "semaphore flags",
-  "telegraph_wheatstonecooke5needle": "needle",
+  "telegraph_wigwag": "us semaphore flags signalcorps",
```
Comment (S-Man42) [inline assets/searchstrings/en.json:553]: Maybe american or unitedstates makes sense here as well?

File: lib/widgets/selector_lists/telegraph_selection.dart [hunk 7]
```diff
@@ -32,7 +32,7 @@ class TelegraphSelection extends GCWSelection {
         className(WigWagSemaphoreTelegraph()),
         className(GaussWeberTelegraph()),
         className(SchillingCanstattTelegraph()),
-        className(WheatstoneCooke5NeedleTelegraph()),
+        className(WheatstoneCookeNeedleTelegraph()),
         className(CCITTPunchTape()),
```
Comment (S-Man42) [inline lib/widgets/selector_lists/telegraph_selection.dart:36]: Is CCITT Punchtape really a candidate for the telegraph section? It's kind of another technology then the others? What about morse?p

File: assets/i18n/en.json (hunk 3 above)
Comment (t-m-z) [reply assets/i18n/en.json:4407]: done

File: assets/searchstrings/common.json (hunk 2 above)
Comment (t-m-z) [reply assets/searchstrings/common.json:542]: yes - however I need a searstring for punchtape - I have to adjust

File: lib/widgets/selector_lists/telegraph_selection.dart (hunk 7 above)
Comment (t-m-z) [reply lib/widgets/selector_lists/telegraph_selection.dart:36]: I think yes beacous of the graphical representation

File: assets/searchstrings/en.json (hunk 6 above)
Comment (t-m-z) [reply assets/searchstrings/en.json:553]: taken

File: assets/searchstrings/de.json (hunk 4 above)
Comment (t-m-z) [reply assets/searchstrings/de.json:509]: done

File: assets/searchstrings/en.json (hunk 5 above)
Comment (t-m-z) [reply assets/searchstrings/en.json:509]: done

File: assets/searchstrings/common.json (hunk 2 above)
Comment (S-Man42) [reply assets/searchstrings/common.json]: maybe punchtape into ccitt of en.json? (if not there, i believe i added it already....)

File: lib/widgets/selector_lists/telegraph_selection.dart (hunk 7 above)
Comment (S-Man42) [reply lib/widgets/selector_lists/telegraph_selection.dart:36]: but a technology (graphical representation) should not be a decision factor for contextual groups... in that case, shadok would fit here as well ;) (polemic, yes, i know... :D) i believe, punchtape is not a classical telegraph as the rest....

File: assets/searchstrings/en.json (hunk 6 above)
Comment (S-Man42) [reply assets/searchstrings/en.json]: could you pleade have a look, in my mind there's another point where you used the us? maybe on wigwag without telegraph or something. (where the question comes up, whether we should reuse such cases....)

**Comment by t-m-z** [comment]
hm - valid point. So it could be a part of the ccitt selection
Sent from MailDroid

-----Original Message-----

File: lib/widgets/selector_lists/telegraph_selection.dart (hunk 7 above)
Comment (S-Man42) [reply lib/widgets/selector_lists/telegraph_selection.dart:36]: i already added it to the ccitt selections ;)

**Comment by t-m-z** [comment]
:-D


Sent from MailDroid

-----Original Message-----

File: assets/searchstrings/en.json (hunk 6 above)
Comment (t-m-z) [reply assets/searchstrings/en.json]: ther was an usa in the german searchstring which is now deleted

**Comment by S-Man42** [comment]
grafik
You need to wrap the punchtape output into a container with a max width. Otherwise on tablets or in the browser it may look very.... funny.

Furthermore you didn't remove the CCITT from telegraph section.

File: assets/i18n/en.json [hunk 8]
```diff
@@ -4428,35 +4428,41 @@
...
   "telegraph_schillingcanstatt_description": "Electrical Telegraph with 1 needle, Code based on the deflection of the needle (left/right)",
   "telegraph_schillingcanstatt_example": "HELLO → rll r lrrr lrrr rlr",
   "telegraph_schillingcanstatt_stop": "STOP",
   "telegraph_schillingcanstatt_goon": "Go ON",
   "telegraph_schillingcanstatt_finish": "FINISH",
-  "telegraph_wheatstonecooke5needle_title": "Wheatstone Cooke Telegraph Code",
-  "telegraph_wheatstonecooke5needle_description": "Electrical Telegraph with 5 needles, Code based on the deflection of five telegraph needles",
-  "telegraph_wheatstonecooke5needle_example": "HELLO → /\\||| /|\\|| |||/\\ |||/\\ ||\\/|",
+  "telegraph_wheatstonecooke_needle_title": "Wheatstone Cooke Telegraph Code",
+  "telegraph_wheatstonecooke_needle_description": "Electrical Telegraph with 1,2,5 needles, Code based on the deflection of telegraph needles",
```
Comment (S-Man42) [inline assets/i18n/en.json:4458]: please change "1,2,5" to "1, 2 or 5" :)

**Comment by S-Man42** [comment]
grafik

Maybe shorten the description text

**Comment by t-m-z** [comment]
> 

at least the line should be shortened; done

File: assets/i18n/en.json (hunk 8 above)
Comment (t-m-z) [reply assets/i18n/en.json:4458]: done

**Comment by S-Man42** [comment]
ok

**Pull request merged**